src/align.cpp

```diff
@@ -2072,7 +2067,7 @@ static void align_oc_msg_colon(chunk_t *so)
    {
       tmp = nas.m_aligned.GetChunk(idx);
 
-      tlen = tmp->str.size();
+      int tlen = tmp->str.size();
```

Comment:
Looks like the type is perhaps `deque<int>::size_type` (from unc_text.h), which is likely, but not guaranteed, `size_t`.